docs/src/PolyhedralGeometry/Polyhedra/pg_polyhedra_constructions.md

```diff
@@ -0,0 +1,88 @@
+```@meta
+CurrentModule = Oscar
+```
+
+```@setup oscar
+using Oscar
+```
+
+```@contents
+Pages = ["pg_polyhedra_constructions.md"]
+```
+
+# Constructions
+
+The standard way to define a polyhedron is by either giving a
+$V$-representation or an $H$-representation.  But polyhedra may also be
+constructed through other means: by name, via operations on other polyhedra, or
+from other objects in Oscar.
+
+## H and V representations
+
+### Intersecting halfspaces: H-representation
+
+Based on the definition of an $H$-representation, the constructor of
+`Polyhedron` can be used:
+
+```@docs
+Polyhedron(A::Union{Oscar.MatElem,AbstractMatrix}, b)
+```
+
+### Computing convex hulls: V-representation
+
+The following defines a polytope, as a `Polyhedron`, via a $V$-representation
+by calling the following function:
+
+```@docs
+convex_hull(::AnyVecOrMat; non_redundant::Bool=false)
+```
+
+This is a standard triangle, defined via a (redundant) $V$-representation  and
+its unique minimal $H$-representation:
+
+```@repl oscar
+T = convex_hull([ 0 0 ; 1 0 ; 0 1; 0 1/2 ])
+facets_as_halfspace_matrix_pair(T)
+```
+
+
+
+
+## Named polyhedra
+For some commonly used types of polyhedra, dedicated constructors are provided:
+
+```@docs
+simplex
+cube
+cross
+archimedean_solid
+```
+
+## Operations on polyhedra
+Polyhedra can be produced through operations on other polyhedra. For example,
+they can be added using Minkowski addition or scaled; each of which results in
+a new polyhedron.
+
+```@docs
++(::Polyhedron, ::Polyhedron)
+*(::Int, ::Polyhedron)
+intersect(::Polyhedron, ::Polyhedron)
+*(::Polyhedron, ::Polyhedron)
+```
+
+The convex hull of two polytopes can be computed via `convex_hull`.
+```@docs
+convex_hull(::Polyhedron,::Polyhedron)
+```
+
+
+
+## Polyhedra from other objects
+
+There is a growing list of functionality for constructing polyhedra coming from
+other mathematical objects. We list a handful below.
```

Comment:
This would be fine for a presentation, a demo, a press release, but not a reference manual. Here, we talk about what is, not what might be (as "growing" does), and we strive to list all relevant functions we provide, not just "a handful".

So I'd remove this.